openhands/server/listen_socket.py

```diff
@@ -24,15 +27,16 @@ async def oh_action(connection_id: str, data: dict):
     # If it's an init, we do it here.
     action = data.get('action', '')
     if action == ActionType.INIT:
-        await openhands_config.github_auth(data)
```

Comment:
I moved this out of here because it is not needed in OSS. (It will be handled separately in SAAS

Reply:
I believe its reason was to avoid some 'if this then that'. If that's not necessary anymore, then great

Reply:
In SAAS this will be handled internally in the implementation of SessionInitStore - no need to pollute the OSS with this here.